recipes/pydm/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pydm" %}
+{% set version = "1.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0274ad8f906116bc03245478585ad596c7b89994a99ea3cd73f990efef6ecaf6
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - pyqt >=5
+    - qtpy
+  run:
+    - python
+    - six
+    - numpy
+    - scipy
+    - pyepics
+    - requests
+    - pyqt >=5
+    - pyqtgraph
+    - qtpy
+
+test:
+  imports:
+    - pydm
+
+about:
+  home: https://github.com/slaclab/pydm 
+  license: SLAC Open
+  license_file: LICENSE.md
```

Comment:
`license_family: OTHER` should be added

Reply:
Addressed with https://github.com/conda-forge/staged-recipes/pull/9639/commits/ea39b6b59d9794be6a30a3f04391e1cd766be7cb